### PR TITLE
LPS-202478 Show the error message when APIKey it is not configured and hide the action if disabled

### PR DESCRIPTION
**Pull request by robertoDiaz**

### CODE REVIEW

File: modules/apps/ai-creator/ai-creator-openai-web/src/main/java/com/liferay/ai/creator/openai/web/internal/display/context/factory/AICreatorOpenAIMenuItemFactoryImpl.java [hunk 1]
```diff
@@ -91,6 +91,27 @@ public MenuItem createAICreatorCreateImageMenuItem(
...
+							return true;
+						}
+					}
+					catch (ConfigurationException configurationException) {
+						if (_log.isDebugEnabled()) {
+							_log.debug(configurationException);
+						}
+					}
+
+					return false;
```
Comment (adolfopa) [inline modules/apps/ai-creator/ai-creator-openai-web/src/main/java/com/liferay/ai/creator/openai/web/internal/display/context/factory/AICreatorOpenAIMenuItemFactoryImpl.java:113]: This looks a bit weird. Why not keep the old "aiCreatorDALLEEnabled" attribute, and make it simply be `isEnabled() && apiKey() != null`?

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/ai-creator/ai-creator-openai-web/src/main/java/com/liferay/ai/creator/openai/web/internal/display/context/factory/AICreatorOpenAIMenuItemFactoryImpl.java (hunk 1 above)
Comment (robertoDiaz) [reply modules/apps/ai-creator/ai-creator-openai-web/src/main/java/com/liferay/ai/creator/openai/web/internal/display/context/factory/AICreatorOpenAIMenuItemFactoryImpl.java:113]: I've follow the same pattern as: 

https://github.com/liferay-lima/liferay-portal/blob/6f29802034e02288cc7bf1ce59e912d7cf715d14/modules/apps/ai-creator/ai-creator-openai-web/src/main/java/com/liferay/ai/creator/openai/web/internal/editor/configuration/AICreatorOpenAIEditorConfigContributor.java#L45

and 

https://github.com/liferay-lima/liferay-portal/blob/6f29802034e02288cc7bf1ce59e912d7cf715d14/modules/apps/ai-creator/ai-creator-openai-web/src/main/java/com/liferay/ai/creator/openai/web/internal/editor/configuration/AICreatorOpenAIEditorConfigContributor.java#L75

Comment (adolfopa) [reply modules/apps/ai-creator/ai-creator-openai-web/src/main/java/com/liferay/ai/creator/openai/web/internal/display/context/factory/AICreatorOpenAIMenuItemFactoryImpl.java:113]: Ok, sounds good then.

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by SylviaLuan** [comment]
ci:forward:force

**Pull request closed without merging**